ember-flight-icons/index.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const flightIconSprite = require('./public/icons/flight-icon-sprite');
+const flightIconSprite = require('@hashicorp/flight-icons/svg-sprite/svg-sprite-module');
```

Comment:
this is awesome! originally wanted to do this in the `use` (have ember-flight-icons consume the sprite from flight-icons), but couldn't figure it out in ember / couldn't use an external CDN link.

my one comment is...
- i don't think the sprite is necessary in `@hashicorp/flight-icons` rn. so arguably, some folks might complain it's increasing the size
  - 0.0.10-rc = 754 kB 
  - 0.0.11-rc = 1.26 MB
- could the sprite just be in `@hashicorp/ember-flight-icons` or was there a reason you didn't want that?
- i think the catalog coming from `@hashicorp/flight-icons` is great!

***

yes it works in cloud-ui since brian's fix: 
- see https://github.com/hashicorp/cloud-ui/pull/1322
- at root, run `yarn && yarn start`, log in with github, see external-link icons on the first page 
- last i knew the repo may have oddities with vercel, where the deploy must be requested, don't know much about it yet

Reply:
>don't think the sprite is necessary in @hashicorp/flight-icons rn. so arguably, some folks might complain it's increasing the size. the real "problem" is that the initial HTML is much bigger now, but for what I know about how modern browsers work today, this should not be an issue.

it's just in a package, and sits in the `node_modules` folder. if it's not used in the code, it will not increase the final bundle size

> could the sprite just be in @hashicorp/ember-flight-icons or was there a reason you didn't want that?

we had two options to import the "sprite module":
- directly as local file in the `ember-flight-icons` (initial version we did yesterday)
- via npm package.json (current version)

@Dhaulagiri preferred the second, so we don't have to do a new release every time we release a new icon (using the `^1.0.0` annotation in the requirement for the package). I am still not convinced of this approach, I would prefer to release anyway a new version of the `ember-flight-icons` every time, because otherwise we have to require to the developers to manually re-run the install of the package every time we release a new version. In any case, with the current implementation, we can update the package.json in `ember-flight-icons` automatically, if we want, so it's just a matter of deciding what is the best option for us.

> i think the catalog coming from @hashicorp/flight-icons is great!

yep! :)